src/cli.rs

```diff
@@ -44,6 +44,11 @@ pub struct InstallOpts {
     /// With this option, espup will skip GCC installation (it will be handled by esp-idf-sys), hence you won't be able to build no_std applications.
     #[arg(short = 's', long)]
     pub std: bool,
+    /// Install esp's own risc-v toolchain build with croostool-ng
+    ///
+    /// Only install this if you don't want to use the systems risc-v toolchain
+    #[arg(short = 'r', long)]
+    pub esp_riscv_gcc: bool,
```

Comment:
```suggestion
    /// Install Espressif RISC-V toolchain built with croostool-ng
    ///
    /// Only install this if you don't want to use the systems RISC-V toolchain
    #[arg(short = 'r', long)]
    pub esp_riscv_gcc: bool,
```
Mind moving it between `default_host` and `export_file`so it's alphabetically ordered? If you accept my suggestion we would also need to update the README.